community/portworx/ibm_cloud_pak/manifest.yaml

```diff
@@ -1,36 +1,36 @@
 charts:
-  - archive: file:portworx-1.0.13.tgz
+  - archive: file:portworx-1.0.14.tgz
     repository-keys:
       - customRegistryURL
 images:
-- image: portworx/oci-monitor:2.2.0.1
+- image: portworx/oci-monitor:2.3.2
```

Comment:
We had discussed earlier to use the image registry `icr.io/ext/portworx` to upload the images and use that in helm charts. I still dont see its used. Please start using this registry as its needed for private clusters. So we will use this as a registry for IBM deployments.

Reply:
Hi Akash, the changes for ICP are being done in a separate PR.

Reply:
@sharma-tapas : I assume you are referring to icr registry. If its taken care in another PR then I am fine here.

Reply:
Yes, its for the ICR and supporting private cluster(s).